### DISCUSSION TITLE
Introduce Yourself  To the Community  -> Web3 World

### DISCUSSION THREAD

**Discussion by Vikash-8090-Yadav** [Vikash-8090-Yadav/Future.WebNet#146] [General]
**Hello, :hand:  my name is Vikash Kumar Yadav and I'm from Varanasi . I am  blockchain / Web3 developr currently  exploring Devop's field , I built many project on the :spider_web:   Web3 , Blockchain , Ether .My teach stack is basically Docker :ship:  , Kub8 , React/Next for the Fronted  and Ether , Solidity ,  Hardhat for the Blockchain part.**  

If u have any doubt's Regarding any project Feel free to ask here in this discussion  group  only .  :ballot_box_with_check: 

 I'm excited to be here and hope to learn more about  Open source / Blockchain  and meet new people.
Let's Connect :world_map: 
All the social media link available  in my Github profile . 

Note : **All please introduce your self in this dsicussion**

**Comment by Syedhussainsaber** [comment]
Hey 🖐️, I am Syed Hussain Saber from Hyderabad, I am a frontend developer also now learning blockchain development. I hope that I can learn many more things by working as a contributor in this project 😊.

**Reply by Vikash-8090-Yadav** [reply]
Great to see u here  .  Thank's for choosing Web3-World . 
Looking forward to see more contribution from your side .

**Comment by SomeshShinde24** [comment]
Hello, I'm Somesh Shinde from Mumbai and I am currently pursuing BE in IT. I have done projects in Domain like ML, IoT, and Web Dev and currently working on an Augmented Reality project along with that I have to start my Open Source Journey with SWOC 3 hope my contributions will enhance the project.

**Reply by Vikash-8090-Yadav** [reply]
Great to see u here . Thank's for choosing Web3-World .
Looking forward to see more contribution from your side .

**Comment by Sanitya7** [comment]
Hi Web 3 Enthusiasts and @Vikash-8090-Yadav 👋 
I am Sanitya Srivastava,
Building Recyteq [ www.recyteq.com ] , a startup, transforming human hair into plant based fertilizer.
I like technology and blockchain is one of the future technologies, with the intent to make an impact I am in SWOC'23 to buidl solutions which can solve the present problems using blockchain.

Currently, I have a basic understanding of C, Python and solidity.
Looking ahead to try my hands out in advanced solidity concepts, truffle, hardhat and ganache-based projects.

**Reply by Vikash-8090-Yadav** [reply]
Thank's for this man . Thanks once again for choosing Web3World once again.  🔥✌️

**Comment by AnishDubey27** [comment]
Hello everyone,
I'm Anish, currently a sophomore in India pursuing Bachelor's of Technology in Computer Science and Engineering.
I am a DevOps enthusiast and currently learning about various tools and technologies related to DevOps.
Last Hacktoberfest got me into open source contributing, looking forward to contribute to  more projects like this one.

You can connect with me here:
Portfolio: https://anishdubey27.github.io
Linkedin: https://linkedin.com/in/anish-dubey
Twitter: https://twitter.com/anish_dubey_
Github: https://github.com/AnishDubey27

**Comment by theanuragshukla** [comment]
Hello everyone, I am Anurag Shukla.
I am a Full stack web and blockchain developer. 
I'm studying Computer Science and Engineering, B.Tech, 2nd year.
I like to participate in Hackathons and build something new using the tech I know.

website: [anuragshukla.xyz](anuragshukla.xyz)

**Comment by arpithams1947** [comment]
Hello, I am Arpitha M S and am from Bangalore. A former web developer and a web3 enthusiast. Currently learning about blockchain and its applications. Thank you for having me and I am excited to learn new things from this journey while contributing to the project. Happy to meet and connect with you all!

**Comment by arpithams1947** [comment]
Hi, This is Arpitha. Happy to connect with you🙂

**Comment by Rishikavishnoi** [comment]
Hello everyone,
I am Rishika Vishnoi, currently pursuing a Bachelor's of Technology in Computer Science and Engineering.
I'm familiar with Web development tools such as MERN stack and also contribute to open source programs such as social summer of code, Hacktoberfest, hackclub RAIT.
Recently, I started my Web 3 journey from Alchemy University and now looking forward to getting more knowledge of the Web 3 world by contributing to this project. 
Let's connect with me:-
LinkedIn: https://www.linkedin.com/in/rishikavishnoi/
GitHub:- https://github.com/Rishikavishnoi
Twitter:- https://twitter.com/vishnoi_rishika

**Reply by Vikash-8090-Yadav** [reply]
Hey @Rishikavishnoi  thank's for joining Solidity-pathshala . Looking fdrward to see contribution from your side . 

:smile:

**Comment by ashutoshumar** [comment]
Hello, I'm Ashutosh Kumar from Bhagalpur and I am currently pursuing Btech in CSE from BCE Bhagalpur. I have done projects in Android and Web Dev and currently learning web3 and Blockchain, going to start my open source journey with SWOC 3, hoping to have a good amount of learning from this project.

**Comment by yagyesh-bobde** [comment]
I am Yagyesh Bobde, currently pursuing BTech from RCOEM, Nagpur. I am passionate about learning and building new things. 
I am familiar with the MERN stack and some basic solidity programming. I would love to learn from and connect with this community.

You can connect with me here: 
Linkedin: https://www.linkedin.com/in/bobde-yagyesh

**Comment by Kaushan-Dutta** [comment]
Hi everyone!! I am Kaushan Dutta and I am currently a 2nd Year student pursuing his BTech in CSE. I am currently working on **Full stack BlockChain Development** . I have been a **MERN Stack Developer** and have participated in a number of hackathons like _Hackobertfest, HackOdisha, Amritathon and so on_. I am looking forward to contribute as much in this event and love to work with everyone of you.
**Connect me :**
Website: _https://kaushan-dutta.vercel.app/_
Linkedin: _https://www.linkedin.com/in/kaushan-dutta-bb68b021a/_

**Comment by Greyshaws** [comment]
Hi everyone. I am Gracious Igwe, a smart contract developer from Nigeria. I am also a final year law student at the University of Nigeria. I have built several projects on the blockchain, and have participated in a number of hackathons. I am currently pursuing a career in smart contract auditing. I am looking forward to learning as much as I can, making substantial contributions to this project, and equally connecting with this community.

LinkedIn: https://www.linkedin.com/in/gracious-i-105a63b8/
Twitter: https://twitter.com/greyshaws

**Comment by nerdy-nakul** [comment]
Hello Everyone,
My name is Nakul Singh. I'm from Delhi and currently persuing B.Tech in Computer Science Engineering from Bhagwan Parshuram Institute of Technology. I am passionate about software development and open source contributions. I am deeply interested in OpenCV. I'm a Web3 and Blockchain enthusiast. I also passionate about creating visually appealling and user friendly designs. I would love to learn  and connect from this community.

Connect Me :
LinkedIn : https://www.linkedin.com/in/nakul-singh-506048205/

**Comment by Abhinavcode13** [comment]
Hello Folks!!
Myself Abhinav Kumar. I'm from Mathura ,UP but currently studying in BMS college of engineering at Bangalore .As a second-year Computer Science and Engineering student at BMSCE, I have gained proficiency in various programming languages such as C++, Java, and Python. In addition, I am well-versed in several technologies including HTML, CSS, JavaScript, React.js, Express.js, MongoDB, Power BI,
MySQL, Flutter and Figma.

I am passionate about blockchain, data science, data analytics, and software development. I enjoy participating in competitive programming competitions and building projects in data science, machine learning, and software development.

Looking forward to connecting with like-minded individuals and exploring new opportunities in the tech industry.

let's connect ❤️ : https://www.linkedin.com/in/abhinav131/
Twitter 🐦 : https://twitter.com/Abhinavxkumar13

**Comment by SayandevSantra** [comment]
Hi everyone!! I am Sayandev Santra , a 3rd Year student pursuing  BTech in CSE. I am currently learning  BlockChain Development . I have  participated in a number of hackathons and this is my first contribution to open source so  I am looking forward to contribute a lot in this event and love to work with everyone .
let's connect ❤️ :https://www.linkedin.com/in/sayandev-santra-b56b5a151/

**Comment by dqwira** [comment]
Hallo Everyone!
My name is Diky Wiraguna, and I am a final-year student in Informatics Engineering. I have been interested in blockchain technology since last year and have been studying it ever since. I hope that by contributing to open-source projects here, I can gain more knowledge especially about web3 and blockchain ecosystem and make new friends.

Follow for Follback : https://github.com/Raguna9

**Comment by Arioum** [comment]
Greetings Everyone!

My name is Amogh Rao. I am a BCA grad, now pursuing MCA 1st year @SrinivasUniversity. I am an aspiring web developer who began his journey a year ago. I am also a freelance web designer and my experience so far is limited to self-study/trial and error (it's a parallel skill that I foster along with web development). 

The journey has been fascinating so far. But I do find myself stagnating every now and then. I hope to learn more by collaborating on open source. As such I look forward to working with you all.

If you wish to connect you shall find my socials in my GitHub profile.

Thank you for @Vikash-8090-Yadav for this opportunity and Thank you all!
